tests/inputs/example_service/test_example_service.py

```diff
@@ -0,0 +1,95 @@
+from typing import AsyncIterator, AsyncIterable
+
+import pytest
+from grpclib.testing import ChannelFor
+
+from tests.output_betterproto.example_service.example_service import (
+    TestBase,
+    TestStub,
+    ExampleRequest,
+    ExampleResponse,
+)
+
+
+class ExampleService(TestBase):
+    async def example_unary_unary(
+        self, example_string: str, example_integer: int
+    ) -> "ExampleResponse":
+        return ExampleResponse(
+            example_string=example_string,
+            example_integer=example_integer,
+        )
+
+    async def example_unary_stream(
+        self, example_string: str, example_integer: int
+    ) -> AsyncIterator["ExampleResponse"]:
+        response = ExampleResponse(
+            example_string=example_string,
+            example_integer=example_integer,
+        )
+        yield response
+        yield response
+        yield response
+
+    async def example_stream_unary(
+        self, example_request_iterator: AsyncIterable["ExampleRequest"]
```

Comment:
Is this argument name still correct, I suspect mypy would complain.

Reply:
Fixed.

I don't know if mypy complains about this specifically, since mypy highlights these entries with
```
syntax error in type comment
```
for me anyway. Since this is also true for the client-facing methods that I didn't change, I just assumed that my mypy config is somewhat broken and paid it no further attention.

Reply:
Yea, there's a bit of work to get the whole project properly typed, but every little helps.